### PR TITLE
Add note for `ronn` command.

### PR DESCRIPTION
**Pull request by overtrue**
if not ronn command:

```sh
ronn -w -s toc -r5 --markdown man/*.ronn
sh: ronn: command not found
rake aborted!
Command failed with status (127): [ronn -w -s toc -r5 --markdown man/*.ronn...]
/Users/overtrue/Protobuf-PHP/Rakefile:123:in `block (2 levels) in <top (required)>'
Tasks: TOP => pear:package => doc:build
(See full trace by running task with --trace)
```

### CODE REVIEW

**Comment by overtrue** [comment]
I signed it!

File: docs/quickstart/php.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ $ cd Protobuf-PHP
 $ rake pear:package version=1.0
 $ [sudo] pear install Protobuf-1.0.tgz
 ```
+> maybe you should install 'ronn' if the command not found:
+> ```sh
+> $ gem install ronn
```
Comment (stanley-cheung) [inline docs/quickstart/php.md]: While we are at it, can you add `rake` as well? So `gem install rake ronn`. Thanks.

File: docs/quickstart/php.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ $ cd Protobuf-PHP
 $ rake pear:package version=1.0
 $ [sudo] pear install Protobuf-1.0.tgz
 ```
+> maybe you should install 'rake' or 'ronn' if they not installed before:
```
Comment (stanley-cheung) [inline docs/quickstart/php.md]: Please change this line to 

"If you did not have 'rake' or 'ronn' installed, please install them using this command:"

Comment (overtrue) [inline docs/quickstart/php.md]: Thank you. 😄

File: docs/quickstart/php.md [hunk 3]
```diff
@@ -71,6 +71,10 @@ $ cd Protobuf-PHP
 $ rake pear:package version=1.0
 $ [sudo] pear install Protobuf-1.0.tgz
 ```
+> If you did not have 'rake' or 'ronn' installed, please install them using this command::
```
Comment (stanley-cheung) [inline docs/quickstart/php.md]: Please remove the extra `:` at the end of the line.

**Pull request closed without merging**